recipe/install-symlinks.sh

```diff
@@ -2,13 +2,19 @@
 
 source ${RECIPE_DIR}/setup_compiler.sh
 
+if [[ "$target_platform" == "win-"* ]]; then
+  symlink="cp"
```

Comment:
haha 😆

Perhaps a slight improvement:
```suggestion
  symlink_or_copy="cp"
```